RELEASE-NOTES.txt

```diff
@@ -1,3 +1,7 @@
+1.21.0
+------
+* [Android] Dark Mode
+
```

Comment:
We should probably target v1.23.0 by now 👍

Reply:
Hey @etoledom, thanks for the feedback.
I just put: "future release" instead of version as we aren't sure when the WPAndroid part will be done. Also, I will update 


> > Link color should be blue 30 (double-check iOS for proper value)
> 
> I noticed the iOS / Android color difference on links:
> 
> iOS	Android
> ![Simulator Screen Shot - iPhone 11 - 2019-12-31 at 12 06 02](https://user-images.githubusercontent.com/40213/71624458-fcacae80-2be2-11ea-98b8-34effabb1f6e.png)	![Screenshot_1577790361](https://user-images.githubusercontent.com/40213/71624462-02a28f80-2be3-11ea-8a4e-51920c32d98f.png)

This should be fixed.